README.md

```diff
@@ -256,6 +256,7 @@ Name | Website | Region
 [Olark](/company-profiles/olark.md) | https://www.olark.com/ | UTC-8 to UTC+1
 [Olist](/company-profiles/olist.md) | https://olist.com/ | Brazil
 [Olo](/company-profiles/olo.md) | https://www.olo.com/ | US
+[OmbuLabs](/company-profiles/ombu-labs.md) | https://www.ombulabs.com/ | UTC-8 to UTC-2 
```

Comment:
Hey @etagwerker, thanks for the PR!

Can we put `Worldwide` here too?  We're in the process of simplifying this field and eventually eliminating it from the readme.  For now it will help to make it the same as what's in the company profile.

For more details about that, see #485.